dev/jsx/weather.jsx

```diff
@@ -23,10 +23,10 @@ class FetchWeather extends React.Component {
   getWeatherData() {
     // Request data from multiple APIs simultaneously
     axios.all([
-      axios.get(`http://api.wunderground.com/api/5332856fca0fe1e7/conditions/q/${this.props.params.state}/${this.props.params.city}.json`),
-      axios.get(`http://api.wunderground.com/api/5332856fca0fe1e7/forecast/q/${this.props.params.state}/${this.props.params.city}.json`),
-      axios.get(`http://api.wunderground.com/api/5332856fca0fe1e7/hourly/q/${this.props.params.state}/${this.props.params.city}.json`),
-      axios.get(`http://api.wunderground.com/api/5332856fca0fe1e7/forecast10day/q/${this.props.params.state}/${this.props.params.city}.json`)
+      axios.get(`https://api.wunderground.com/api/5332856fca0fe1e7/conditions/q/${this.props.params.state}/${this.props.params.city}.json`),
```

Comment:
I made this change on the branch after this ("foreign" branch). Should I still fix it here or just leave it since the next branch has this change?